pkg/logger/logger.go

```diff
@@ -132,13 +132,19 @@ func (l *Logger) Output(calldepth int, message string) {
 	l.mu.Lock()
 	defer l.mu.Unlock()
 
-	l.stdLogTemplate.Execute(l.writer, stdLogMessageData{
+	err := l.stdLogTemplate.Execute(l.writer, stdLogMessageData{
```

Comment:
We should make sure that this doesn't throw errors if a field is missing or not all of the fields are used. We don't want to panic on custom formats unless they're particularly wrong I would guess 🤔

Reply:
This was the area I was most torn about. I landed on this: since we are an authentication system, I feel like we have to uphold audit-ability of the CIA security triad and exit if authentication logs would be wrong/missing.

You bring up a good point about the template. I'll double check to confirm we validate that in the validator already.  I think when I dug into the errors this could raise last night, it mostly related to io.Write issues to the handle.

Reply:
I think panicking if we can't write the log makes sense, but I want that to be panicking because the disk is full rather than panicking if the user misconfigured the logging format if that makes sense